README.md

```diff
@@ -70,6 +70,10 @@ Generate output as-path access-list.
 
 Host running IRRD database (default: `rr.ntt.net`).
 
+#### -I
+
+Disable identification of client to server (not recommended).
```

Comment:
I don't see a compelling reason to offer this feature. It is truly terribly to debug anything on the server side without this hint.